content/en/docs/instrumentation/python/exporters.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Exporters
+weight: 4
+---
+
+In order to visualize and analyze your traces and metrics, you will need to
+export them to a backend.
```

Comment:
It's a bit confusing to be told to send data to a backend and then start with the console exporter

Reply:
Removed the backend bit from the intro sentence. Since this follows other articles, we could change it to move the console exporter but that'd likely be a more uniform change across all languages.